docroot/modules/custom/webform_template/tests/src/Kernel/WebformTemplateControllerTest.php

```diff
@@ -12,7 +12,7 @@
  *
  * @package Drupal\Tests\webform_template\Kernel
  */
-class RequestTemplateTest extends KernelTestBase {
+class WebformTemplateControllerTest extends KernelTestBase {
```

Comment:
You've changed the name of a class. Are you sure this is correct?

Reply:
@brockfanning I changed the correct class name by looking into the suggestion